README.md

```diff
@@ -1 +1,50 @@
-# ramen
+# Ramen
+
+Ramen is a **disaster-recovery orchestrator** for stateful applications across
+a set of peer kubernetes clusters that,
+
+- Preserve state using persistent volumes provisioned using [ceph-csi](https://github.com/ceph/ceph-csi/)
+- Uses the CSI [storage replication extensions](https://github.com/csi-addons/volume-replication-operator)
+  , which enable ceph-csi to replicate state across peer clusters
+- Are deployed and managed using [open-cluster-management](https://open-cluster-management.io/concepts/architecture/)
+
+Ramen provides **cloud-native interfaces** to orchestrate the life-cycle of an
+applications state across peer clusters. These include,
+
+- Failing over an applications state to a peer cluster on unavailability of
+  the currently working cluster
+- Migration of an applications state to a peer cluster
+
+**NOTE:** Application deployment life cycle is managed as part of OCM
+orchestrators, and Ramen concerns itself with the application state only.
```

Comment:
Should it be "OCM orchestrator" instead of "OCM orchestrators"?

Reply:
I was going plural as there are many OCM orchestrators that work to make it happen.

Also, on hindsight, IMO we can remove this NOTE as well, thoughts?